bindinfo/bind_cache.go

```diff
@@ -75,26 +76,30 @@ func (c *bindCache) get(key bindCacheKey) []*BindRecord {
 
 // set inserts an item to the cache. It's not thread-safe.
 // Only other functions of the bindCache can use this function.
-func (c *bindCache) set(key bindCacheKey, value []*BindRecord) bool {
+// The set operation will return error message when the memory usage of binding_cache exceeds its capacity.
+func (c *bindCache) set(key bindCacheKey, value []*BindRecord) (ok bool, err error) {
 	mem := calcBindCacheKVMem(key, value)
 	if mem > c.memCapacity { // ignore this kv pair if its size is too large
-		return false
+		err = errors.New("The memory usage of the binding_cache exceeds its capacity")
```

Comment:
I think it's a bit strange here. Actually we haven't exceeded the capacity yet here, because we didn't put this binding into the cache